src/routes/ukraine/+page.svelte

```diff
@@ -19,17 +19,18 @@
 	const UN_DONATION_LINK = 'https://crisisrelief.un.org/t/ukraine' + SRC_EXTERNAL;
 	const U24_DONATION_LINK = 'https://u24.gov.ua/donate' + SRC_EXTERNAL;
 
-	const warStartedIso = '2022-02-24T00:00:00-02:00';
+	const warStartedIso = '2022-02-24T00:05:00-02:00';
 	const warStarted = new Date(warStartedIso).getTime();
 	const now = new Date().getTime();
 	const warDurationMs = now - warStarted;
 	let elapsedSeconds = 0;
 
 	let isDirectDonation = false;
 
-	console.log('V', warStarted, now, warDurationMs);
-
-	setInterval(() => elapsedSeconds++, 1000);
+	setInterval(() => {
+		elapsedSeconds++;
+		elapsedSeconds %= 60;
```

Comment:
I don't think this is necessary on L:55 you see  
`(warDurationMs + elapsedSeconds * 1000)`

So the input into the formatting function is in MS, so `70 seconds` => warDurationMs + 70 * 1000, so it does not make a difference.

Reply:
lol, it's not reproducible anymore. Just checked the logoi.dev

Not sure what happened. But I did reproduce it 2w ago

Reply:
I fixed it in https://github.com/logoidev/logoidev/commit/1c6ddbc5ace871a145422e3afc28c78875bdf047